hypothesis-python/src/hypothesis/searchstrategy/misc.py

```diff
@@ -87,7 +87,7 @@ class SampledFromStrategy(SearchStrategy):
 
     def __init__(self, elements):
         SearchStrategy.__init__(self)
-        self.elements = d.check_sample(elements)
+        self.elements = d.check_sample(elements, SampledFromStrategy.__name__)
```

Comment:
The name here can just be the string `'sampled_from'`, as that's the user-facing API.  Similar notes for the other calls.